openmc/mgxs/mgxs.py

```diff
@@ -149,6 +149,8 @@ class MGXS(object):
         Name of the multi-group cross section
     rxn_type : str
         Reaction type (e.g., 'total', 'nu-fission', etc.)
+    nu : bool
```

Comment:
My reasoning is:

1.  I figured a user could be expected to think that `FissionXS` had a `nu` attribute that `NuFissionMatrixXS` would as well and so they would eventually try and get to it.

2. I agree that for others it makes little physical sense (or confusing, what does nu mean in `CaptureXS`?), but in those cases I've removed it from the docstring so its sort of invisible and basically just baggage from the `MGXS` Abstract

3. It avoided me having to do the properties/setters for every class that needed it! And I'm lazy! ;-)